search-commons/src/test/java/no/unit/nva/search/SearchClientTest.java

```diff
@@ -466,6 +465,114 @@ void searchTicketsResponseShouldFormatAggregationsCorrectly() throws IOException
+            listAllExistingRulesForDoiRequests(sentRequest);
+        var mustExistsTicketStatusInDoiRequests =
+            rulesForIncludingDoiRequests.stream().anyMatch(condition -> condition.fieldName().equals("status"));
+        var mustExistsOrganizationIdsInDoiRequests =
+            rulesForIncludingDoiRequests.stream()
+                .anyMatch(condition -> condition.fieldName().equals("organizationIds"));
+        assertTrue(mustExistsTicketStatusInDoiRequests, "Could not find rule for including ticket status for "
+                                                        + "DoiRequests");
+        assertTrue(mustExistsOrganizationIdsInDoiRequests,
+                   "Could not find rule for including organizationIds for DoiRequests");
+    }
+
+    @Test
+    void shouldSendQueryWithAllExistingRulesForPublicationConversationTypeWhenSearchingForTickets()
+        throws ApiGatewayException, IOException {
+        var mockSearchResponse = generateMockSearchResponse(NO_HITS_RESPONSE_JSON);
+
+        AtomicReference<SearchRequest> sentRequestBuffer = new AtomicReference<>();
+        var restClientWrapper = new RestHighLevelClientWrapper((RestHighLevelClient) null) {
+            @Override
+            public SearchResponse search(SearchRequest searchRequest, RequestOptions requestOptions) {
+                sentRequestBuffer.set(searchRequest);
+                return mockSearchResponse;
+            }
+        };
+
+        var searchClient = new SearchClient(restClientWrapper, cachedJwtProvider);
+        SearchTicketsQuery searchTicketsQuery = new SearchTicketsQuery(DEFAULT_PAGE_SIZE, DEFAULT_PAGE_NO,
+                                                                       emptyList());
+        searchClient.findTicketsForOrganizationIds(generateSampleViewingScope(),
+                                                   searchTicketsQuery,
+                                                   OPENSEARCH_TICKET_ENDPOINT_INDEX);
+        var sentRequest = sentRequestBuffer.get();
+
+        var rulesForIncludingExistingPublicationConversation =
+            listAllExistingRulesForPublicationConversation(sentRequest);
+
+        var mustExistsTicketStatusInDoiRequests =
+            rulesForIncludingExistingPublicationConversation.stream()
+                .anyMatch(condition -> condition.fieldName().equals("status"));
+        var mustExistsOrganizationIdsInDoiRequests =
+            rulesForIncludingExistingPublicationConversation.stream()
+                .anyMatch(condition -> condition.fieldName().equals(
+                    "organizationIds"));
+        assertTrue(mustExistsTicketStatusInDoiRequests, "Could not find rule for including ticket status for "
+                                                        + "GeneralSupportCase");
+        assertTrue(mustExistsOrganizationIdsInDoiRequests,
+                   "Could not find rule for including OrganizationIds for GeneralSupportCase");
+    }
```

Comment:
```
        var rulesForIncludingPublicationConversation = extractQueryBuilderValues(sentRequest);

        assertThat(rulesForIncludingPublicationConversation, hasItem(GENERAL_SUPPORT_CASE));
    }

    private List<Object> extractQueryBuilderValues(SearchRequest sentRequest) {
        return listAllInclusionAndExclusionRulesForPublicationConversation(sentRequest)
                .stream().map(MatchQueryBuilder::value)
                .collect(Collectors.toList());
    }
```

Reply:
> Have a look at comments from @brinxmat

@ketilaa fixed the comments from @brinxmat. Waiting for RE-request revivew.